tdrs-backend/tdpservice/scheduling/db_backup.py

```diff
@@ -94,7 +94,16 @@ def backup_database(file_name,
     pg_dump -F c --no-acl --no-owner -f backup.pg postgresql://${USERNAME}:${PASSWORD}@${HOST}:${PORT}/${NAME}
     """
     try:
-        cmd = postgres_client + "pg_dump -Fc --no-acl -f " + file_name + " -d " + database_uri
+        # TODO: This is a bandaid until the correct logic is determined for the system values
+        # cmd = postgres_client + "pg_dump -Fc --no-acl -f " + file_name + " -d " + database_uri
+        db_host = settings.DATABASES['default']['HOST']
+        db_port = settings.DATABASES['default']['PORT']
+        db_name = settings.DATABASES['default']['NAME']
+        db_user = settings.DATABASES['default']['USER']
+
+        export_password = f"export PGPASSWORD={settings.DATABASES['default']['PASSWORD']}"
```

Comment:
I have been doing the re-parsing and it always fails at the db_backup step. Happy to pair on this